src/networking.c

```diff
@@ -3374,6 +3377,17 @@ NULL
         } else {
             addReplyArrayLen(c,0);
         }
+    } else if (!strcasecmp(c->argv[1]->ptr, "no-touch") && c->argc == 3) {
```

Comment:
nowdays we don't need to check `argc` in subcommand, just like #11331.
```suggestion
    } else if (!strcasecmp(c->argv[1]->ptr, "no-touch")) {
```